platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/Activator.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.filter.websso;
+
+import org.ops4j.pax.web.extender.whiteboard.runtime.DefaultSecurityConfigurationMapping;
+import org.ops4j.pax.web.service.whiteboard.SecurityConfigurationMapping;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+
+public class Activator implements BundleActivator {
```

Comment:
This was placed fairly arbitrarily so can probably go somewhere better. 
Previously we were able to do this via config but those config values aren't supported anymore. So using this bundle activator we can set the auth method and realm name we need for triggering out JettyAuthenticator